biolink-model.yaml

```diff
@@ -6145,6 +6145,717 @@ slots:
+      (pointwise) mutual information in collocation extraction. Proceedings of GSCL,
+      30, 31-40. https://svn.spraakdata.gu.se/repos/gerlof/pub/www/Docs/npmi-pfd.pdf
+    range: string
+
+  mutual dependence:
+    is_a: association slot
+    description: >-
+      Mutual Dependency (MD), often also known as PMI\xB2, is a variant
+      of PMI that attempts to overcome PMI's preference for rare events. For details,
+      see Thanopoulos, A., Fakotakis, N., & Kokkinakis, G. (2002, May). Comparative
+      Evaluation of Collocation Extraction Metrics. In LREC (Vol. 2, pp. 620-625).
+      http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.8101&rep=rep1&type=pdf
+    range: string
+
+  log frequency biased mutual dependence:
+    is_a: association slot
+    description: >-
+      Log-Frequency-Biased Mutual Dependency is a variant of Mutual Dependency
+      that results in favoring the more frequently occuring pairs when ranking. For
+      details, see Thanopoulos, A., Fakotakis, N., & Kokkinakis, G. (2002, May). Comparative
+      Evaluation of Collocation Extraction Metrics. In LREC (Vol. 2, pp. 620-625).
+      http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.8101&rep=rep1&type=pdf
+    range: string
+
+  tmkp concept1 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept1) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept2 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept2) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept cooccurrence count:
+    is_a: association slot
+    description: The raw count of observations of two concepts cooccurring in the
+      same document (or part of a document) in a text corpus.
+    range: string
+
+  contribution towards patient survival:
+    is_a: association slot
+    description: >-
+      Contribution is a summary statistic indicating the level of contribution
+      an entity has on patient survival time as measured by its quantifiable impact
+      on a distribution of patient survival time.
+    range: integer
+    examples:
+      - value: '[-1,1]: e.g. -0.23, 1, 0.75'
+
+  probability of prolonged survival:
+    is_a: association slot
+    description: Reflects the posterior distribution for likliehood of survival past
+      a specified period of time, given some evidence established in the connecting
+      nodes.
+    range: integer
+    examples:
+      - value: 'e.g., .57, 1, .01'
+
+  half maximal inhibitory concentration:
+    is_a: association slot
+    description: IC50 indicates how much drug is needed to inhibit a biological process
+      by half, thus providing a measure of potency of an antagonist drug in pharmacological
+      research
+    range: datetime
+    examples:
+      - value: '-2.112848714'
+
+  binding constant:
```

Comment:
We can add mappings here https://bioportal.bioontology.org/search?utf8=%E2%9C%93&query=binding+constant&button=